src/main/java/com/google/api/generator/gapic/composer/common/AbstractTransportServiceStubClassComposer.java

```diff
@@ -931,14 +1000,43 @@ private Expr createCallableInitExpr(
     return javaMethods;
   }
 
+  private boolean checkOperationPollingMethod(Service service) {
```

Comment:
Honestly, I prefer not using static methods, if not necessary, as they are much less flexible. For private methods it doesn't even matter that much, but even if a method is not using any class state it might in the future or subclasses may be using it, also static methods often cause troubles during testing as they harder to mock, if necessary.